ndmg/stats/qa_tractography.py

```diff
@@ -0,0 +1,113 @@
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+
+from fury import actor, window, colormap, ui
+from dipy.tracking.streamline import Streamlines
+from PIL import Image
+from matplotlib.colors import LinearSegmentedColormap
+
+def combine_plot(qa_out_path,brain_path):
+    """
+    Combine the nine plots produced by qa_tractography function into one image.
+
+    Parameters
+    ----------
+    qa_out_path : str
+        Path for the output QA imgae file(s)
+    brain_path : str
+        Path for the reference brain scan, in order to get the scan volume.
+    """
+    
+    brain_ref = nib.load(brain_path)
+    brain_ref_shape = brain_ref.shape[0:3]
+
+#   plt.rcParams.update({"axes.labelsize":"x-large","axes.titlesize":"x-large"})
+    plots_list = ['imgage1000000.png','imgage1000001.png','imgage1000002.png',
+                 'imgage2000000.png','imgage2000001.png','imgage2000002.png',
+                 'imgage3000000.png','imgage3000001.png','imgage3000002.png']
```

Comment:
Will the created images always be named this? Is there any way to set the image filenames?

Reply:
The function I used to create the nine images is fury.window in dipy. I totally create nine images but three of them are a pair. I can only change the out_path name, for example, instead of image1, image2, image3, I can change them to more meaningful names like Rotate_Z_axis, Rotate_Y_axis, Rotate_X_axis. But the following numbers (i.e. 000000, 000001, 00002 is produced by the function itself. I can't change them.

Reply:
https://github.com/neurodata/ndmg/pull/375/commits/f9edd4a64206da4025ac3ece5970c649b197ff73